cli/internal/packagemanager/yarn.go

```diff
@@ -42,6 +48,13 @@ var nodejsYarn = PackageManager{
 
 		globs, err := pm.getWorkspaceGlobs(rootpath)
 		if err != nil {
+			// In case of a non-monorepo, the workspaces field is empty and only node_modules in the root should be ignored
+			_, ok := err.(*NoWorkspacesFoundError)
+			if ok {
```

Comment:
you can use `errors.As()` here instead of coercing the type. Example from another part of the codebase:
https://github.com/vercel/turbo/blob/6bc8b5e0bdf920e799ca693bf697acdf0796640b/cli/internal/core/engine.go#L183-L193